### ISSUE TITLE
price data fetch

### ISSUE THREAD

**Issue by ravi19821982** [AlphaWallet/alpha-wallet-android#3135]
is any way that when we add our token in wallet lock assets   and can we fetch our token price from  pancake swap  also price not show of some populer token like bnb and eth  how to solve that

**Comment by ravi19821982** [comment]
![price](https://user-images.githubusercontent.com/126206377/222923692-45b8b3d4-bcde-419e-95a8-c63fd557a0a6.jpg)

**Comment by seabornlee** [comment]
Thank you for reporting this issue. Have you already fixed it, or should I look into it soon?

**Comment by ravi19821982** [comment]
no i cant can you help me in this

**Comment by seabornlee** [comment]
You can view the price chart by clicking on the token. The number you mentioned indicates the amount of change, and since your balance is 0, all of the values are also 0.

**Comment by ravi19821982** [comment]
bro i just want to confirm that we are fetching data from coingicko right
now so os any want that i can fetch data from pancakswap ?

**Comment by seabornlee** [comment]
Yes, you can look into https://github.com/AlphaWallet/alpha-wallet-android/blob/cf609cb8a5c80a40e3fa9950ae524015db09681e/app/src/main/java/com/alphawallet/app/service/TickerService.java
